v3/archerkeyboard/desire65/desire65.json

```diff
@@ -0,0 +1,167 @@
+{
+  "name": "desire65",
+  "vendorId": "0x0361",
+  "productId": "0x0002",
+  "matrix": {
+    "rows": 5,
+    "cols": 16
+  },
+  "keycodes": [
+    "qmk_lighting"
+  ],
+  "menus": [
+    "qmk_rgblight"
+  ],
```

Comment:
@akyp This enables all animations, just making sure that it is what you want. It looks like QMK source only enables a handful and would require a custom menu to show the correct amount.

To have the correct options shown, you should add the custom menu RGB Light explicit declaration and only use the effects you have enabled by removing the unused ones from the option list: https://www.caniusevia.com/docs/built_in_menus#qmk_backlight_rgblight

Reply:
Changed and new JSON tested. Thanks for the reminder.